source/unified-test-format/unified-test-format.rst

```diff
@@ -624,6 +624,7 @@ The structure of this object is as follows:
     specifications:
 
     - `Causal Consistency <../causal-consistency/causal-consistency.rst#sessionoptions-changes>`__
+    - `Snapshot Reads <../sessions/snapshot-sessions.rst#sessionoptions-changes>`__
```

Comment:
I forgot to mention this, but you should add a make target for session tests in the [unified test runner Makefile](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/tests/Makefile). This ensures all unified session tests are checked against the schema during the CI build.

Reply:
Great, thanks! Added, and will verify when yml tests are added.